e2e/specs/graphviz_chart.spec.ts

```diff
@@ -26,14 +26,20 @@ describe("st.graphviz_chart", () => {
   });
 
   it("displays a graph with two connected nodes", () => {
-    cy.get("#graphviz-chart-0 svg").matchImageSnapshot("graphviz-chart-0");
+    cy.get("#graphviz-chart-0 svg").then(el =>
```

Comment:
I may be misunderstanding something, but I don't think this new pattern does anything meaningfully different, does it? All cypress return values are already promises (and the Cypress internals retry each `get` multiple times until it succeeds).

Unless I'm missing something, I think this should be reverted. (If I _am_ missing something, can you add a comment in the code explaining why the `then` pattern is needed here?)